junit-jupiter-params/src/test/java/org/junit/jupiter/params/ParameterizedTestExtensionTests.java

```diff
@@ -92,8 +92,8 @@ void throwsExceptionWhenParameterizedTestIsNotInvokedAtLeastOnce() {
 
 		Stream<TestTemplateInvocationContext> stream = this.parameterizedTestExtension.provideTestTemplateInvocationContexts(
 			extensionContextWithAnnotatedTestMethod);
-		//cause the stream to be evaluated
-		stream.toArray();
+		// cause the stream to be evaluated
+		Object[] unused = stream.toArray();
```

Comment:
Hmmmm.... aren't we now going to see warnings about the `unused` variable being _unused_?